README.md

```diff
@@ -849,183 +976,179 @@ platformOptions.preSpawnProcessConfigurator = { _, fileAttr in
-        }
-
-        /// `ConsoleBehavior` defines how should the window appear
-        /// when spawning a new process
-        public struct WindowStyle: Sendable, Hashable {
-            /// Activates and displays a window of normal size
-            public static let normal: Self
-            /// Does not activate a new window
-            public static let hidden: Self
-            /// Activates the window and displays it as a maximized window.
-            public static let maximized: Self
-            /// Activates the window and displays it as a minimized window.
-            public static let minimized: Self
-        }
-
-        /// Sets user info when starting the process. If this
-        /// property is set, the Subprocess will be run
-        /// as the provided user
-        public var userCredentials: UserCredentials? = nil
-        /// The console behavior of the new process,
-        /// default to inheriting the console from parent process
-        public var consoleBehavior: ConsoleBehavior = .inherit
-        /// Window style to use when the process is started
-        public var windowStyle: WindowStyle = .normal
-        /// Whether to create a new process group for the new
-        /// process. The process group includes all processes
-        /// that are descendants of this root process.
-        /// The process identifier of the new process group
-        /// is the same as the process identifier.
-        public var createProcessGroup: Bool = false
-        /// A closure to configure platform-specific
-        /// spawning constructs. This closure enables direct
-        /// configuration or override of underlying platform-specific
-        /// spawn settings that `Subprocess` utilizes internally,
-        /// in cases where Subprocess does not provide higher-level
-        /// APIs for such modifications.
-        ///
-        /// On Windows, Subprocess uses `CreateProcessW()` as the
-        /// underlying spawning mechanism. This closure allows
-        /// modification of the `dwCreationFlags` creation flag
-        /// and startup info `STARTUPINFOW` before
-        /// they are sent to `CreateProcessW()`.
-        public var preSpawnProcessConfigurator: (
-            @Sendable (
-                inout DWORD,
-                inout STARTUPINFOW
-            ) throws -> Void
-        )? = nil
-
-        public init() {}
+/// The collection of platform-specific settings
+/// to configure the subprocess when running
+@available(FoundationPreview 0.4, *)
+@available(iOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+public struct PlatformOptions: Sendable, Hashable {
+    public struct UserCredentials: Sendable, Hashable {
+        // The name of the user. This is the name
+        // of the user account to run as.
+        public var username: String
+        // The clear-text password for the account.
+        public var password: String
```

Comment:
I assume this mirrors an underlying Win32 API?

Reply:
Yes. As bad as it is the Win32 API literately says "plain text password"... 😅